Makefile

```diff
@@ -1,71 +1,76 @@
 # Copyright 2021 NetApp, Inc.  All Rights Reserved
+.DEFAULT_GOAL:=help
+SHELL:=/bin/bash
```

Comment:
Not sure that path will always exist - isn't SHELL := bash sufficient?

Reply:
I think we can remove it all together. Doesn't make any diff

Reply:
removed it.